### DISCUSSION TITLE
【案出し】リポジトリ(サイト)のコンセプト・ストーリーなど

### DISCUSSION THREAD

**Discussion by kazzyfrog** [first-contributions-ja/first-contributions-ja.github.io#122] [00: 未分類]
V2に向けて、思いついたアイデアを投稿・発散させるスレッドです！

- コンセプト・ストーリー
- 個別ページやトップページの表示
- Contributors.jsonや画像など、入力してもらう内容

## 関連
#112 
#119

**Comment by kazzyfrog** [comment]
アイデア：

- myOctcatの画像を使う

> - 絵文字と色のストーリー
> - POP，激しい、レインボー、ごちゃごちゃ
> - POPは楽しく取り組めたらいいな
> - 激しい：色々なものがあっていい、おもしろい
> - 自分の色
> https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/76#discussioncomment-8892381

- 自分の個性をアピール
- 楽しむ

**Comment by kazzyfrog** [comment]
複数の絵文字で、自分を表現するとかもありかも！

下記は、６つの絵文字でジブリ映画を表している；
https://www.pinterest.jp/pin/on-twitter--663647695104918659/

**Comment by kazzyfrog** [comment]
以下のサイトのヒーローセクションのように、
色＋一言＋図形（種類の選択式？）で、ステッカー作成してもいいかも！

https://www.mydesignbuddy.co.uk/?ref=landingfolio

**Comment by kazzyfrog** [comment]
オリジナルのLGTMの、メッセージやら、画像（ステッカー）やら、コードブロックが作れる！とかでもいいかも
たまに、LGTMって書かれた画像を使って貼り付ける人や、プロジェクトのノリがあったりするので！

オリジナルLGTMジェネレータ
例：
```
🎉🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🎉
  _        ____   _____   __  __ 
 | |      / ___| |_   _| |  \/  |
 | |     | |  _    | |   | |\/| |
 | |___  | |_| |   | |   | |  | |
 |_____|  \____|   |_|   |_|  |_|
                                 
🎉🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🎉
```

**Reply by kazzyfrog** [reply]
一言メッセージとか入れてもいいかも！
by handle nameとかも

**Reply by kazzyfrog** [reply]
でも、これはオリジナルLGTMジェネレータという別のアプリになってしまうかも。。

**Comment by kazzyfrog** [comment]
絵文字は、３つくらいあったほうがいいかも？？

**Comment by pss-aileen** [comment]
## jsonに「theme」を追加して、表示の仕方を変更できるようにする

jsonに

"theme": "normal"

の項目を追加して、個別ページのテーマを選んでもらう形式に
メンテナーが好きなテーマを作成することが可能

```json
{
  "name": "あなたのハンドルネーム",
  "github": "あなたのGitHubアカウントのURL",
  "favoriteStack": "あなたの好きなプログラミング言語・技術スタック",
  "favoriteColor": "あなたの好きなカラーコード",
  "favoriteEmoji": "あなたの好きな絵文字",
  "theme": "normal"
},
```

- normalテーマは私たちで用意
- どうせなので、3人で別々のテーマを作る（以下例）
  - pop-theme
  - simple-theme
  - type-theme
  - Aileen-theme
  - kazzyfrog-theme
  - kagome-theme
個別ページの表示方法を選べる
テンプレート、テーマ

これをすると npm run dev でプレビューを確認する作業がきちんと必要だったり
コントリビューターとして参加したい人がいればテーマを作ることもできるのでNext.jsの勉強もでき、コントリビュートもできて一石二鳥。勉強になる？

**Reply by pss-aileen** [reply]
ただ、ここまでいくと確認することとかが逆に多すぎてハードル高くなってしまうかも、どうかなー

**Reply by kagomen** [reply]
themeが選択できると楽しそうですね！🍭🌟

themeは必須ではなくオプションとして、記述すれば適用される・記述しなければdefaultのpop-themeが適用される、みたいにしても良いかも？と思いました
READMEでもステップ外のオプション設定として、themeの設定について説明された別mdをリンクする、という形だったら、参加者のハードルは高くないかな？と

v2.0.0ではとりあえず導入せずに、v.2.1.0以降で実装していく、という方針だったら運営側的にもハードルはそこまで高くないかな？と思いました！

どうなんでしょうか❓

**Reply by pss-aileen** [reply]
> themeは必須ではなくオプションとして、記述すれば適用される・記述しなければdefaultのpop-themeが適用される、みたいにしても良いかも？と思いました

まさしくそんな感じです！
defaultを基本記述してもらっておく感じで

2.0.0以降で自分たちがテーマをそれぞれ1つ作って選択肢を作ったり、
コントリビューター？メンテナーが案を出してくれてテーマを作ってくれたら楽しいのかな〜！と
練習、勉強になりそうだなぁと！

> READMEでもステップ外のオプション設定として、themeの設定について説明された別mdをリンクする、という形だったら、参加者のハードルは高くないかな？と

そうですね、defaultを普通としておいて、こだわりたい方、ちょっと頑張ってみたい、表示まで意識をまわせる人には別で案内ができると良いのかもしれません！そうすればハードル上がらない...！！

**Comment by kagomen** [comment]
ここまで話が進んでいる中ぶっこむ感じで申し訳ないのですが、こちらのプロジェクトに参加してくれる人ってそもそも「OSSの体験をしたい」人で、そういう人たちに対して、好きなカラーコードや好きな絵文字を情報として要求するのって、どうなんだろう❓（妥当性や必然性がないのでは❓）という気がしてきました……🤔
そして、参加したことで個人ページが作成されることは、参加者のメリットになるのかなあ？という気もしてきました

だったら、コントリビューションした人たちを一覧で出して、その中に自分の名前が追加される方が、「コントリビューション」っぽい（＝OSSに参加した感がある）のではないかなーと……

そもそも、他の初心者向けOSS体験プロジェクトと差別化したい、ということで話し合いが始まったと思いますが、日本人の初心者が日本語で安心してOSSの体験ができる、というだけでちゃんと差別化されている気がしました。

どうでしょうか。みなさんのご意見をお聞きしたいです！！！👂

**Reply by pss-aileen** [reply]
> ここまで話が進んでいる中ぶっこむ感じで申し訳ないのですが、

問題提起ありがとうございます！！🙌

先に個人的な結論、意見をまとめると

- パブリックな目的: 日本語OSS体験リポジトリの提供
- リポジトリのポリシー: 楽しく参加
- プライベートな目的: 共同プロジェクトを体験、学習した技術を使用

上記を考慮して
**日本語であることに加え、楽しい要素（絵文字やカラーコードの要求）があっても良いのかなと思いました。**

以下、詳細です。

---

> こちらのプロジェクトに参加してくれる人ってそもそも「OSSの体験をしたい」人で、そういう人たちに対して、好きなカラーコードや好きな絵文字を情報として要求するのって、どうなんだろう❓（妥当性や必然性がないのでは❓）という気がしてきました……🤔

**リポジトリのポリシーを軸にすると、要求することは妥当、あっても良いかなと思います。**

---

> そして、参加したことで個人ページが作成されることは、参加者のメリットになるのかなあ？という気もしてきました

- 私たち運営側のメリット
  - 特徴的であることで、拡散して広めてもらえる可能性がある
  - 自分たちが共同でアイディアをあつめて作った実績となる
- 参加者のメリット
  - **視覚的に楽しい・面白い部分があることで、達成感を得られる**
  - 個人体験）自分が他のOSS体験リポジトリに参加した時、HTML記述タイプでしたが、自分が作ったものがマージされて表示されたのは嬉しい&楽しかった

---

> だったら、コントリビューションした人たちを一覧で出して、その中に自分の名前が追加される方が、「コントリビューション」っぽい（＝OSSに参加した感がある）のではないかなーと……

- これも良いと思います！
- ただ、せっかくなので、**コンセプトや特徴があると惹かれる一つの要素になったり、人に認知してもらいやすくなるのではないか？と思います。**
- 自分が参加した https://github.com/Syknapse/Contribute-To-This-Project は視覚的な面白さとHTMLがかければOKだったので、これが参加の決め手になりました

---

> そもそも、他の初心者向けOSS体験プロジェクトと差別化したい、ということで話し合いが始まったと思いますが、日本人の初心者が日本語で安心してOSSの体験ができる、というだけでちゃんと差別化されている気がしました。

> 他の初心者向けOSS体験プロジェクトと差別化したい

このような流れでしたっけ...記憶喪失で...泣

私の記憶だと、
**日本語プロジェクトを作ることが最初のそもそものテーマであって、差別化...はすでにされている**と思います。

ただ、日本語ということですすめているけれど、
コンセプトが定まらないとデザインも、見せ方、表示させ方も決まらない〜、特徴がない、ということで悩み、
現在は楽しい・ポップを軸（仮決定）として、
それらをどのようにページに表示されるかで悩んでいる...といった感じかな...と。

私の記憶と違った場合ですが、以下のように考えます。

- リポジトリ・パブリックな目的: 完全日本語のOSSプロジェクトを体験を提供すること
- 他のリポジトリとの差別化ポイント
  - READMEやWebサイトが日本語であり、日本語で質問が可能なこと

日本語であることを重点とするのであれば、
Webサイトを作ることを前提とするのであれば
サイトはHTML、CSSで作成。
HTMLに名前を書いていってもらう...といったシンプルな形か。

それとも、宣伝としてのWebページはつくらず、
READMEに全てを集約し、markdownファイルに名前を足していってもらうか...といった方にして
日本語で丁寧な解説をつけることが、ユーザーには良いのかなと思います。

---

上記も踏まえた上ですが
プライベートな（私たちの）目的としては
共同プロジェクトを体験するといったものもあったかと思います。（あと技術も...？）

その他、個人的ですが作業しながら、
このリポジトリの試行錯誤をみんなが見れるようにすることで、
「どんなふうにプロジェクトが作られるのか」を残せる超絶貴重？？な日本語リポジトリになれたらいいな〜〜〜と思っていました。笑

あと、せっかくNext.jsいれているので、
それを活かした何かが作れれば技術の勉強にもなるからよいかな...と！

**Reply by pss-aileen** [reply]
超長くてすみません😭

**Reply by kagomen** [reply]
@pass-aileen
ご意見ありがとうございます！😭
お忙しい中なのに……すみません！💦

以下、自分なりに考えたことです！

***

目的やポリシーなどを再確認するのは確かに大事ですね。！
パブリックな目的とプライベートな目的は、同じ認識です。
その中で、リポジトリのポリシーがあまり掴めないのですが、どこかで決定しましたっけ……？
もし「楽しく参加」をポリシーに掲げるとなると、コンセプトの追加によって楽しく参加してもらうことはかなり大変なことだと思うので、もっと深く考えるべきかなと思いました……！

また、ポリシーが「楽しく参加」ではなく、「画面を楽しげにする」であれば、絵文字の使用は良いと思います！
画面のキャッチーさは、

> 特徴的であることで、拡散して広めてもらえる可能性がある

こちらに繋がりますよね。

そして、「画面を楽しげにする」のであれば、やはり個別ページは必要ないのではないか、と思いました。
自分の想像力が乏しいだけかもしれませんが、あまりキャッチーになる想像がつかなくて……。（個別ページがあることによって、拡散力が上がるとも思えず……）

その代わりに、トップページを作り込むのはどうでしょうか。？
例えば、

- トップページに選んでもらった絵文字を降らす

上記の現在の案に加えて、

- 降ってきた絵文字にカーソルを合わせると、絵文字のそばに吹き出し💬が表示される
- 吹き出しの中に入力してもらった個人の情報が表示される（名前と一言挨拶とか）

など。
単純に一人の情報より密度が増しますし、

> OSSに参加した感

も感じてもらえるかな？と思いました。
この他にも、「画面を楽しげ（キャッチー）にする」（->なぜなら拡散して広めてもらえる可能性があるから）をポリシーとして決定したら、色々と案が出てきそうだなと思いました。！

**Reply by pss-aileen** [reply]
## ポリシー、雰囲気について
以下のところで仮決定かと思いっていました...！🙇‍♀️
ご確認ください！

楽しく参加というのは、画面が楽しい感じというのも含みます。
すごい広い意味なのですが...参加の流れが楽しいというより、反映された結果が楽しいというか。それに画面の楽しさに繋がる感じです。

https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/122#discussioncomment-8899986

この考えをベースにストーリー性を決める...と思っていました。
例えば今はポップとか楽しげに参加するといったふわっとした感じなので

「日本語のOSS参加チュートリアルを作成にあたり
ただ、テキストを追加するだけでなく
ユーザーに絵文字やカラーコードを入力してもらい
それを反映させて、楽しくコントリビューションに参加できることを目標（？）とし作成しました」的な感じ......のコンセプト、ストーリー

https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/136#discussioncomment-8952745

## 個別ページのあり、なしについて

結論あってもなくても良いと思います

ちなみにですが、個別ページを作る以前に、ある程度TOPページを作るこむことは前提として考えています！
正直TOPを作り込んだら個別もなしで良い可能性がありますが...

そしてそもそも個別ページについては以下のような感じです。

## そもそもの個別ページの必要性や、なぜ作る方向になったのか

- **最初の最初の案で一覧ページに個人情報載せたくない人がいるのではないか？→個別ページを作ろう的な流れがあった**、気がする..？
- そこから個別ページを作る...？かな？といった雰囲気に
- **コンセプト等色々すすまないので仮決定で作ろうという方向へ**
- その流れと、個人的にはコントリビュートに参加して、個別のページが生成されたら「なんかできた！」とシェアしたくなるタイプということもあったので、SNSにシェアしてもらえるきっかけになるのでは？と考えていました

そんな感じで、**確固たる「絶対に必要だ！」という気持ちはないのでなしでも良いと思います！**

## 個別ページありなしも含め、デザイン案を出すところでつめていければ...

カードや個別ページは「仮」として決定している段階なので
カードや個別ページでなくてもよいですし、
色々な表現方法があると思います。

とりあえず、こちらの件も含め
まだ具体的にどうするかが決定していないので
案出しということで、こちらのDiscussionsで引き続きアイディアを出し、

最終的に案としてデザインカンプ的なものを作り、
良いところをあつめる...
といった感じにできればよいのではないかと考えていましたが...どうでしょうか？

https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/136#discussioncomment-8949812

ただ、contributors.jsonの内容は確定させないとバージョン1が出せないので、
そこに記載してある情報を前提にTOPページ、個別ページの有無等を考える必要はあると思います。

---

とりあえず、最初の疑問に立ち返って
絵文字やカラーコードを入力してもらうことは、現状のポリシー、方向性としてはあって良いと思っています。

それをふまえて個別ページやカードをどうするかは、ここで案出しの中でもう少し時間がかけて考えられれば...と思っています！

**Reply by pss-aileen** [reply]
自分が言語化がちゃんとできておらず
色々認識がずれているような気がしているので
どこかで上手に認識をあわせられたら...と思います！

ちょっとまた時間があるタイミングで...！

**Reply by kazzyfrog** [reply]
お二人ともありがとうございます🙌

パブリックな目的とプライベートな目的は、自分も同じ認識なので、全員で共有できているかと思います！

リポジトリのポリシーに関しても、
Aileenさんの投稿の「楽しく参加」と、kagomeさんの投稿の「画面を楽しげにする」は、
表現が違うだけで、同じことを指しているんだと思います！
これに関しても、自分も同じ認識なので、全員で共有できているかと思います！

なので、お二人が言うように、絵文字の使用は良いと思います！👍
厳密にはこれも、まだ確定ではないが、現状その方向性で考えよう！という認識です。
それも踏まえて、このスレッドでアイデアを出して、リポジトリのポリシー（コンセプト）も具体的に確定させたいと思ってます！

> 「画面を楽しげ（キャッチー）にする」（->なぜなら拡散して広めてもらえる可能性があるから）をポリシーとして決定したら、色々と案が出てきそうだなと思いました。！

自分はその認識で、このスレッドで案を出していければと思っていました！
（それに関しては、自分が色々擦り合わせられていない状態で、このスレッドを作ってしまってたなと思いました。。🙏）

なので、
> こちらのDiscussionsで引き続きアイディアを出し、
最終的に案としてデザインカンプ的なものを作り、
良いところをあつめる...
といった感じにできればよいのではないかと考えていましたが...どうでしょうか？

こちらで良いと思います！！

**Reply by kazzyfrog** [reply]
## 一旦まとめ
現状こんな感じ
- カラーコード、絵文字などのContributors.jsonに入力してもらう内容は、確定じゃないのでこのスレッドでアイデア出して決める
- トップページ、個別ページの有無・内容に関しても、確定じゃないのでこのスレッドでアイデア出して決める

そもそも、最初に確定していたことは、
「OSSプロジェクトへの**コントリビュート**を完全日本語で体験できる」ということ。
→これはおそらく、このリポジトリに対する全員の共通認識！

## その上で、じゃあ**具体的に**「どんなコントリビュートができるか？」という点について。

「コントリビュートを体験できる」ということだけを実現するなら、
変更を加えてマージができれば良いんだと思います！
（usernameの記入だけとかでもいいし、サイトに表示させなくてもいい）

ただ、本当に文字通りコントリビュートという作業が体験できるというよりかは、
「**工夫**」を加えることで、もっと良いチュートリアルになると思っています！

「**工夫**」というのが、何かしら意味のある内容を入力してもらうことで、それがサイトに表示されるということ。
それによって、チュートリアル完了の達成感とか、視覚的な楽しさ、満足感が生まれて、
拡散してもらえる可能性も上がるかもです！
同時に、サイトが賑やかだと、参加のモチベーションも上がり、新たなユーザーに参加してもらえる可能性？も上がるのかなと思ってます！

### 要は、「コントリビュートの手順を体験できる」ことに振り切るか、独自の工夫をするか。
このプロジェクトのメリットは、「OSSの体験ができること」というのはその通りだと思います！
ただそれは、やらないと分からないことかも？とも思っています！

なので、多分、少しでも参加のモチベーションを高めるような、
何かしらの気になる要素（POPなデザインや、他の人がチュートリアルを完了したことをシェアしている、、など）
があったほうがいいのかな？と個人的には思ってます！

## 工夫する方向で行くなら、「何かしら意味のある内容を入力してもらう」必要がある！
その意味のある内容って具体的になに？
というのを、このスレッドで決めたい！という認識でした。
（現状だと、絵文字とか、好きな色とかで、仮決定？みたいになっているやつを、確定させたい！）

言い換えれば、
- サイトのコンセプト（上記の表現で言うならリポジトリのポリシー）が決まれば、
- Contributors.jsonに入力してもらう内容も決まり、
- サイトのデザインカンプも作れるようになる！

という感じですね！

なので、
> 好きなカラーコードや好きな絵文字を情報として要求するのって、どうなんだろう❓
妥当性や必然性がない

という指摘は、**現状**その通りだと思います！
なので、このスレッドでコンセプトとそれに基づいて、入力してもらう内容を決めよう！という認識です😎

そしたら、コンセプトと、入力してもらう内容と、サイトのデザインに一貫性が生まれ、
ただのコントリビューション体験ではなく、このプロジェクト独自の世界観？にラップされた、
他とは違った体験ができるのかな？と思った次第です！

**Reply by kazzyfrog** [reply]
**あと、個別ページに関しても、どっちでも大丈夫です！**

現状は、コンセプトやら、Contributors.jsonに入力してもらう内容やらが決まった後に、
トップページや個別ページの有無、デザインを詰められればと思ってます！

個別ページは、自分が最初に言ったアイデアですが、
ユーザーに入力してもらった内容が、何かしらの形でトップページに反映される？なら、
それをTwitter上で見れた方が、映えるかなと思った感じでした。

トップページのURLは、シェアした時に、サイトのOG画像ですが、
個別ページならzennやqiitaの記事のように、OGイメージをページ毎に生成できるので、、！

入力してもらった内容で視覚的な楽しさを作り出す + Twitterなどで共有してもらいたい
なら、そうしてもいいかもなーというアイデアでした！

個別ページは、全然確定事項ではないという認識だったので、なくてもOKです！
まあそれも、デザインを決める段階で、色々アイデア出しながら決めればと思います！👍👍

**Reply by kazzyfrog** [reply]
そもそも、「OSSコントリビュート体験」に、どれくらいニーズかあるかどうかわからないので、
学習したいこととか、作りたいサイトベースで、進めても良さそう！！

**Reply by kagomen** [reply]
お二人とも、ありがとうございます！😭

自分の中で考えの整理・言語化ができていなかったのですが、こちらの議論のなかでなんとなく自分の考えも明確になってきたので、最初にもう一度改めて自分の意見を伝えさせてください！（もしかしたら最初の主張から意見が変わっている点もあると思いますが、ご了承ください💦）

***

## そもそもの立場
- MTGで絵文字と色の案が出たとき、良いなと思った！
- MTGから時間が経ち、これってどうなんだろう……？と思ってきた

そして以下は、デザインの提案ではなく、もっと手前の、

- 案が固まってきてる中の、軌道修正としての意見
- **コンセプトの考え方**を共有するための**具体例**

として捉えていただければと思います。

## 個別ページの生成について
- OSSの参加の結果として、**反対**のことが生じている
    -> パブリックなものに参加したのに、個人のページが作成されるという謎な状態 
- 単純に、アイデアとして安易では？と思う

## トップページについて
- 現在の、絵文字が降ってくる、という案はキャッチーで良いと思う
- パブリックなものに参加したという結果としても、表現として成立している
- ただ、自分が「変更を加えた感じ」が薄い、わかりにくい
- 絵文字に加えて、カーソルをホバーしたら名前も表示される、とか、そもそも絵文字と一緒に名前も表示されるなどとしたら、「そこに自分が参加した感じ」が得られる

といった感じです。
このデザイン案を採用してほしい！ということではなく、この考え方についてどう思うか？（賛成か反対か、反対ならなぜ反対なのか？）という発言です。

***

> 本当に文字通りコントリビュートという作業が体験できるというよりかは、
「工夫」を加えることで、もっと良いチュートリアルになると思っています

これは超同意です！！！
ただ、その工夫が付け焼き刃のものであるなら工夫はしなくても良い、というより**しない方が良い**というのが自分の正直な意見です。
そして上記に述べた意見は、付け焼き刃を避けてより良い案を出すための、つまり軌道修正をするための、自分なりの意見でした。

ただ、

> そもそも、「OSSコントリビュート体験」に、どれくらいニーズかあるかどうかわからないので、
学習したいこととか、作りたいサイトベースで、進めても良さそう！！

こちらでおっしゃってるように、自分たちの学習や作りたいものを優先するのであれば、そこまで考えることでもないと思います！

なので自分の結論としましては、

**目的（複数可）とそれらの優先順位について考えたい**

という感じです！
ただ、勝手に自分は

> 案が固まってきてる

と思っていたので上記のような結論に至りましたが、
お二人的にはまだ現在は案出し＝ブレーンストーミングの状態なのかな？（特に @kazzyfrog さんはそうかな？）と思いましたので、自分の意見は的外れだった気もします。

また、

> トップページのURLは、シェアした時に、サイトのOG画像ですが、 個別ページならzennやqiitaの記事のように、OGイメージをページ毎に生成できるので、、！

こちらすっかり忘れていました！💦
OG画像が必要かどうか、なぜ必要なのか、についても意見を出し合えたら良いなと思いましたー！

長くなってしまって、すみません🙏💦

**Reply by kagomen** [reply]
追記です！

> Aileenさんの投稿の「楽しく参加」と、kagomeさんの投稿の「画面を楽しげにする」は、
表現が違うだけで、同じことを指しているんだと思います！

こちらは、プロジェクトの画面を楽しげにすることは、本当に参加者が楽しく参加することに繋がるのか？という疑問の提示でした。
少なくとも自分は繋がらない派なので、（コンセプトによって）楽しく参加をさせたいのであればもっと全く別の角度からアイデアを考える必要があるのでは？と思った次第でした！

コンセプトを現状のままに、「楽しく参加」を考えるのであれば、一例として、

**参加者よりもメンテナーの方が楽しい -> 参加者をメンテナー側に連れてくるように仕向ける**

みたいな角度から考えを進めた方が良いのではないか？という感じです！

ただこちらも、

> そもそも、「OSSコントリビュート体験」に、どれくらいニーズかあるかどうかわからないので、
学習したいこととか、作りたいサイトベースで、進めても良さそう！！

こちらを優先するのであれば、特段考える必要はないかな！と思っています！

**Reply by kagomen** [reply]
## [Figma会議](https://www.figma.com/file/xTVaHLu2OeeIBS8nGeDAvC/%E8%80%83%E3%81%88%E3%81%A6%E3%81%84%E3%82%8B%E3%81%93%E3%81%A8%E3%81%AE%E3%81%BE%E3%81%A8%E3%82%81?type=whiteboard&node-id=0%3A1&t=x0Ax8cAy7MEYwM2U-1)のまとめ

- 現時点の焦点は、ドッグフーディングで見つかった修正をすること
  - それができればv1.0.0で区切る！
  - その後に、リファクタリングとv2の仕様を決める必要がある

- 当スレッドは「v2の仕様を決める」ための判断材料やアイデアを自由に投稿していく場所
  - アイデア出しの段階で、コンセプトはまだ決まっていない

- このプロジェクトの目的
  - 共同プロジェクト自体を（楽しく）体験する
  - 学んだ技術を使う！学習の実践であり、成果物作り！
  - とはいえ、それが誰かの役に立ったりして実際に使ってもらえるなら、それに越したことはない。なので、負荷がかかりすぎない範囲で、ペルソナの設定やユーザー体験も考えよう！

- このプロジェクトの区切りは4月末〜5月くらい（暫定）
  - 実装の進み具合やユーザーからの反響次第で前後する可能性あり

### その他
- ドッグフーディング後の修正案についての話し合いは、Figmaでやってもいいかも？

**Comment by pss-aileen** [comment]
@kagomen @kazzyfrog 
こちらの件、Figma上でおはなししつつ、
どこかでMTGできたら良いのかなと思って招待してます！ご確認ください！

https://www.figma.com/file/xTVaHLu2OeeIBS8nGeDAvC/%E8%80%83%E3%81%88%E3%81%A6%E3%81%84%E3%82%8B%E3%81%93%E3%81%A8%E3%81%AE%E3%81%BE%E3%81%A8%E3%82%81?type=whiteboard&node-id=0%3A1&t=x0Ax8cAy7MEYwM2U-1
